CHANGELOG.md

```diff
@@ -0,0 +1,24 @@
+<!--
+SPDX-FileCopyrightText: 2023 SPDX contributors
+
+SPDX-License-Identifier: CC0-1.0
+-->
+
+# Change Log
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+### Added
+- Support for [snippet information] (https://spdx.github.io/spdx-spec/v2.3/snippet-information/).
+- Support for [annotation information] (https://spdx.github.io/spdx-spec/v2.3/annotations/).
+- Support for [other licensing information] (https://spdx.github.io/spdx-spec/v2.3/other-licensing-information-detected/).
+- Support for [SPDX 3] (https://github.com/spdx/spdx-3-model).
+
+## [0.1.0] - 2023-11-27
```

Comment:
do you want to publish and tag today?

Reply:
Actually, we could also start with 0.0.1 today and plan release 0.1.0 for the end of the week.